src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml

```diff
@@ -1,14 +1,55 @@
 NWBFile:
-  related_publications:
-    https://doi.org/### or link to APA or MLA citation of the publication
-  session_description:
-    A rich text description of the experiment. Can also just be the abstract of the publication.
+  keywords: 
+    - auditory cortex
+    - predictive coding
+    - optogenetics
+  experiment_description:
+    Identifying mistakes is important for improving performance during acoustic behaviors like speech and musicianship. Although hearing is instrumental for monitoring and adapting these behaviors, the neural circuits that integrate motor, acoustic, and goal-related signals to detect errors and guide ongoing sensorimotor adaptation in mammals remain unidentified. Here, we develop a novel closed-loop, sound-guided behavior that requires mice to use real-time acoustic feedback to guide skilled ongoing forelimb movements. Large scale electrophysiology recordings reveal that the mouse auditory cortex integrates information about sound and movement, as well as encodes error- and learning-related signals during this sound-generating behavior. Distinct groups of auditory cortex neurons signal different error types, and the activity of these neurons predicts both within-trial and across-trial behavioral adaptations. Brief, behavior-triggered optogenetic suppression of auditory cortex during error signaling hinders behavioral corrections on both rapid and long time scales, indicating that cortical error signals are necessary for skilled acoustic behaviors. Together, these experiments identify a cortical role for detecting errors and learning from mistakes and suggest that the auditory cortex plays a critical role in skilled, sound-generating behavior in mammals.
   institution: Institution where the lab is located
   lab: Schneider
   experimenter:
-    - Last, First Middle
-    - Last, First Middle
+    - Zempolich, Grant W.
+    - Schneider, David M.
 Subject:
-  species: Rattus norvegicus
-  age: TBD  # in ISO 8601, such as "P1W2D"
-  sex: TBD  # One of M, F, U, or O
+  species: Mus musculus
+  age: P12W/  # in ISO 8601, such as "P1W2D"
+  sex: U  # One of M, F, U, or O
+  description: 12-week-old C57BL/6 or VGATChR2-EYFP mice (see Aims approaches below for details) will be used for all behavioral, electrophysiology, and optogenetic experiments. In the VGAT-ChR2-EYFP mouse line, channelrhodopsin (ChR2) is coupled to the vesicular GABA transporter, inducing expression in GABAergic inhibitory neurons ubiquitously across cortex and allowing for real time optogenetic inhibition of brain regions of interest.
+  strain: C57BL/6
+
+Ecephys:
+  Device:
+    - name: DeviceEcephys
```

Comment:
What do you think of having a more descriptive name for this?

Do you know which one of the probes listed here is the one the yare using:

https://masmanidislab.neurobio.ucla.edu/images/microprobe_info.pdf

Reply:
Yes, it's 128AxN.